src/test/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealServiceTest.java

```diff
@@ -134,6 +138,8 @@ public void setUp() {
 
         given(emailService.generateUniqueEmailId(any(Appellant.class))).willReturn("Bloggs_33C");
 
+        given(verifyTokenService.verifyTokenSignature(any())).willReturn(true);
+
```

Comment:
It might be good to verify in some test that the verify service is being called correctly too.